app/src/main/java/com/vero/tasky/core/presentation/navigation/RootNavigation.kt

```diff
@@ -22,11 +27,76 @@ fun RootNavigation(
     ) {
         composable(route = Screens.Agenda.route) {
             AgendaScreen(
-                openEventScreen = { },
-                openReminderScreen = { },
-                openTaskScreen = { }
+                openAgendaItemScreen = { agendaItem, isEditable ->
+                    val route = when(agendaItem) {
+                        is AgendaItem.Event -> Screens.Event.route
+                        is AgendaItem.Reminder -> Screens.Reminder.route
+                        is AgendaItem.Task -> Screens.Task.route
+                    }
+                    val parameters = "?itemId=${agendaItem.id}&isEditable=$isEditable"
+                    navController.navigate("$route$parameters")
+                },
+                openNewAgendaItemScreen = { agendaItemType ->
+                    val route = when (agendaItemType) {
+                        AgendaItemType.REMINDER -> Screens.Reminder.route
+                        AgendaItemType.TASK -> Screens.Task.route
+                        else -> Screens.Event.route
+                    }
+                    navController.navigate(route)
+                }
             )
         }
+        composable(
+            route = Screens.Event.route + "?itemId={itemId}&isEditable={isEditable}",
```

Comment:
These parameters (itemId and isEditable) are being reused many times

You should have a constant or similar so if the parameter name changes in the future, you can just change it in one place